DQMOffline/MuonDPG/interface/BaseTnPEfficiencyTask.h

```diff
@@ -0,0 +1,108 @@
+#ifndef BaseTnPEfficiencyTask_H
+#define BaseTnPEfficiencyTask_H
```

Comment:
Normally the full path is preferred for the header guards:
```suggestion
#ifndef DQMOffline_MuonDPG_BaseTnPEfficiencyTask_H
#define DQMOffline_MuonDPG_BaseTnPEfficiencyTask_H
```
(and the same everywhere else)

Reply:
Adding the full path in the next commit